fatiando/gridder.py

```diff
@@ -394,3 +395,55 @@ def cut(x, y, scalars, area):
               if x[i] >= xmin and x[i] <= xmax
               and y[i] >= ymin and y[i] <= ymax]
     return [x[inside], y[inside], [s[inside] for s in scalars]]
+
+
+def cut_regular(x, y, scalars, shape, area):
+    """
+    Return a subsection of a regular grid.
+
+    The returned subsection is not a copy! In technical terms, returns a slice
+    of the numpy arrays. So changes made to the subsection reflect on the
+    original grid. Use numpy.copy to make copies of the subsections and avoid
+    this.
```

Comment:
This is a great docstring!

Could you a `copy` optional argument to the function so that `copy==True` returns a copy and `copy==False` returns a slice? This might be useful if you need to process the slice but don't want to mess with the original data. Putting the option in the function means writing the `numpy.copy` code only once.